ui/scripts/movie.js

```diff
@@ -127,25 +127,21 @@ export default class Movie {
     getFlags(countries, name) {
         let flags = "";
 
-        if (countries.length !== 0) {
-            const list = document.createElement("ul");
-            list.classList.add(name);
-
-            countries.map(country => {
-                if (country.flag) {
-                    flags += `<li class="country-flag">${country.flag}<span class="country-name-tooltip">${country.name}</span></li>`;
-                } else {
-                    // handle null flag
-                    // &#127987; adds white flag symbol
-                    flags += `<li class="country-flag">&#127987;<span class="country-name-tooltip">${country.name}</span></li>`;
-                }
-            })
-            list.innerHTML = flags;
-            return list;
-        } else {
+        if (countries.length == 0) {
             return flags;
```

Comment:
nit: it's usually better to initialize variables as late as possible. For example, you don't need `flags` here, you could just return "". And then initialize `flags` later, below the `if` block.